src/modules/esl-alert/core/esl-alert.ts

```diff
@@ -0,0 +1,85 @@
+import {ExportNs} from '../../esl-utils/environment/export-ns';
+import {bind} from '../../esl-utils/decorators/bind';
+import {attr, jsonAttr} from '../../esl-base-element/core';
+import {ESLBasePopup, PopupActionParams} from '../../esl-base-popup/core';
+import {DeviceDetector} from '../../esl-utils/environment/device-detector';
+
+export interface AlertActionParams extends PopupActionParams {
+  text: string;
+  cls?: string;
+}
+
+@ExportNs('Alert')
+export class ESLAlert extends ESLBasePopup {
+  static is = 'esl-alert';
+  static eventNs = 'esl:alert';
+
+  @jsonAttr<AlertActionParams>({defaultValue: {text: '', cls: ''}})
+  public defaultParams: AlertActionParams;
+
+  @attr({defaultValue: ''}) public cls: string;
+  protected text: HTMLElement;
+
+  public static init() {
+    if (document.querySelector(ESLAlert.is)) return;
+    ESLAlert.register();
+    const alert = document.createElement(ESLAlert.is) as ESLAlert;
+    document.body.appendChild(alert);
+  }
+
+  @bind
+  onTriggerHandler(e: CustomEvent) {
+    if (e.type === `${ESLAlert.eventNs}:show`) {
+      const params = Object.assign({}, {hideDelay: 2500}, e.detail, {force: true});
```

Comment:
Magic number in the method is not good, mb we can introduce "Default" constant somewhere?